pkg/controller/pcideviceclaim/pcideviceclaim_controller.go

```diff
@@ -134,20 +140,44 @@ func loadVfioDrivers() {
 }
 
 func bindDeviceToVFIOPCIDriver(pd *v1beta1.PCIDevice) error {
-	logrus.Infof("Binding device %s vfio-pci", pd.Status.Address)
+	if deviceBoundToDriver(vfioPCIDriverPath, pd.Status.Address) {
+		return nil
+	}
+
+	vendorId := pd.Status.VendorId
+	deviceId := pd.Status.DeviceId
+	var id string = fmt.Sprintf("%s %s", vendorId, deviceId)
+	logrus.Infof("Binding device %s [%s] to vfio-pci", pd.Name, id)
+
+	newIdFile, err := os.OpenFile("/sys/bus/pci/drivers/vfio-pci/new_id", os.O_WRONLY, 0200)
+	if err != nil {
+		logrus.Errorf("Error opening new_id file")
+		return err
```

Comment:
can we just return the error with the above message since the controller should print the message directly?